intro-HPC/ch_account.tex

```diff
@@ -350,7 +350,7 @@ \subsection{Users of the \association}
 You will have to provide a public \emph{ssh} key generated as described
 above. Please attach your public key (i.e., the file named ``id\_rsa.pub''),
 which you will normally find in your .ssh subdirectory within your HOME
-Directory. (i.e., /Users/$<$username$>$/.ssh/id\_rsa.pub).
+Directory. (i.e., /Users/<{}username>{}/.ssh/id\_rsa.pub).
```

Comment:
@redfast00 Why not use `$USER` here? Also, should be in `\verb|...|`?

Reply:
This section is wrapped in an ifantwerpen, and the instructions seem to be only for Mac (even though it isn't wrapped in an ifmac). Is this still correct for antwerpen?:
```
All users (researchers, academic staff, etc.) from the higher education
institutions associated with \university can get a VSC account via the \university. There is not
yet an automated form to request your personal VSC account.

Please e-mail the \hpc staff to get an account (see Contacts information).
```

Reply:
That must be outdated... @sbecuwe, @backelj, @klust?